subsys/caf/modules/ble_adv.c

```diff
@@ -557,6 +557,10 @@ static bool handle_ble_peer_operation_event(const struct ble_peer_operation_even
 
 		err = ble_adv_stop();
 
+		if (err) {
+			break;
```

Comment:
this changes logic - we need to be very careful with such changes (are we sure that all return values from stop are bad? maybe adv was already stopped for some reason).

If we expect no problem should have ever happen here (which seems the case) - let's set the state of module to error (see line 616).

Reply:
btw - move int declaration to the place of usage - that is to line 560 (int err = ble_adv_stop;)

Reply:
We don't use then err at all. I will remove it.